packages/EtendoHookBinder/src/api/sales-order.api.ts

```diff
@@ -1,14 +1,17 @@
-import axiosPrivate from ".";
-import { Models } from "../Models";
+import { Client } from './client';
+import { API_BASE_URL } from './constants';
 
+const axiosPrivate = new Client(API_BASE_URL);
 
 export const pageMetadata = async (): Promise<Models> => {
-  return axiosPrivate.get<Models>('/sws/view').then((res) => res.data);
+  return axiosPrivate.get('/sws/view').then(res => res.data);
 };
 
 export const dataSet = (page: number, size: number): Promise<unknown> => {
-  return axiosPrivate.post('/etendo/org.openbravo.service.datasource/Order', {
-    params: { page, size },
-  } );
+  return axiosPrivate.post(
```

Comment:
There should not be a specific sales order api file. This should be obtained from the metadata of the specific tab. In this case the level 0 tab is the one that will return the url of the dataset. Also note that it should never be hardcoded /etendo since context.name is an environment variable.